app/src/main/java/com/hedvig/app/feature/insurance/ui/InsuranceFragment.kt

```diff
@@ -1,52 +1,73 @@
-package com.hedvig.app.feature.dashboard.ui
+package com.hedvig.app.feature.insurance.ui
 
 import android.os.Bundle
 import android.view.View
 import androidx.fragment.app.Fragment
-import com.hedvig.android.owldroid.graphql.DashboardQuery
+import com.hedvig.android.owldroid.graphql.InsuranceQuery
 import com.hedvig.android.owldroid.graphql.PayinStatusQuery
 import com.hedvig.app.R
-import com.hedvig.app.feature.dashboard.service.DashboardTracker
+import com.hedvig.app.feature.insurance.service.InsuranceTracker
 import com.hedvig.app.feature.loggedin.ui.LoggedInViewModel
 import com.hedvig.app.util.extensions.observe
 import com.hedvig.app.util.extensions.view.remove
-import com.hedvig.app.util.extensions.view.setupToolbarScrollListener
+import com.hedvig.app.util.extensions.view.setupToolbarAlphaScrollListener
 import com.hedvig.app.util.extensions.view.updatePadding
-import kotlinx.android.synthetic.main.fragment_dashboard.*
+import com.hedvig.app.util.getToolbarBarHeight
+import dev.chrisbanes.insetter.doOnApplyWindowInsets
+import kotlinx.android.synthetic.main.fragment_insurance.*
```

Comment:
👮 

As we're transitioning to using `ViewBinding` for view interaction, lets stop using synthetics here.